index.ftd

```diff
@@ -1,51 +1,1617 @@
+role: $inherited.types.button-small
+color: $inherited.colors.text
+link: $link-group.cta
+opacity: 0.6
+style: bold
+
+-- end: ftd.column
+
+-- ftd.column:
+width: fill-container
+children: $link-group.wrapper
+margin-left.rem: 1.8
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: link-group
+
+
+
+
+
+
+
+
+
+
+
+
+-- component render-toc:
+pr.toc-item list toc-obj:
+
+-- ftd.column:
+min-width.fixed.percent: 20
+min-height.fixed.calc: 100vh - 0px
+height: fill-container
+width.fixed.percent: 28.5
+padding-horizontal.px: 26
+padding-top.px: 50
+spacing.fixed.rem: 1.5
+
+-- toc-instance:
+$loop$: $render-toc.toc-obj as $obj
+toc: $obj
+
+-- end: ftd.column
+
+-- end: render-toc
+
+
+
+
+
+
+
+
+
+
+-- component toc-instance:
+pr.toc-item toc:
+pr.toc-item $toc-clone: *$toc-instance.toc
+
+-- ftd.column:
+width: fill-container
+padding-left.px: 14
+
+-- ftd.row:
+width: fill-container
+spacing.fixed.px: 15
+$on-click$: $ftd.toggle($a = $toc-instance.toc-clone.is-open)
+
+-- ftd.image:
+src: $assets.files.static.right.svg
+src if { !toc-instance.toc-clone.is-open }: $assets.files.static.down.svg
+align-self: center
+
+-- ftd.text: $toc-instance.toc.title
+link if { toc-instance.toc.url != NULL }: $toc-instance.toc.url
+role: $inherited.types.button-medium
+color: $inherited.colors.text
+width: fill-container
+
+-- end: ftd.row
+
+-- ftd.column:
+if: { !ftd.is_empty(toc-instance.toc.children) && toc-instance.toc-clone.is-open }
+margin-left.px: 44
+padding-left.px: 10
+margin-top.rem: 2.25
+padding-top.rem: 0
+spacing.fixed.rem: 1.75
+
+-- childrens:
+$loop$: $toc-instance.toc.children as $obj
+toc: $obj
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: toc-instance
+
+
+
+
+
+
+
+
+
+
+-- component childrens:
+pr.toc-item toc:
+
+-- ftd.column:
+width: fill-container
+
+-- ftd.text:
+link if { childrens.toc.url != NULL }: $childrens.toc.url
+text: $childrens.toc.title
+role: $fonts.button-medium
+color: $inherited.colors.text
+opacity: 0.60
+
+-- ftd.column:
+if: {!ftd.is_empty(childrens.toc.children) }
+width: fill-container
+margin-top.px: 24
+spacing.fixed.px: 24
+
+-- child-of-childrens:
+$loop$: $childrens.toc.children as $obj
+toc: $obj
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: childrens
+
+
+
+
+
+
+
+
+
+
+-- component child-of-childrens:
+pr.toc-item toc:
+
+-- ftd.column:
+width: fill-container
+padding-left.px: 10
+spacing.fixed.px: 24
+
+-- ftd.text:
+link if { child-of-childrens.toc.url != NULL }: $child-of-childrens.toc.url
+text: $child-of-childrens.toc.title
+role: $inherited.types.copy-small
+color: $inherited.colors.text
+color if { child-of-childrens.toc.is-active }: $inherited.colors.text-strong
+
+-- ftd.column:
+if: { !ftd.is_empty(child-of-childrens.toc.children) }
+width: fill-container
+
+-- child-of-childrens:
+$loop$: $child-of-childrens.toc.children as $obj
+toc: $obj
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: child-of-childrens
+
+
+
+
+
+
+
+
+
+
+-- component h1:
```

Comment:
@MeenuKumari28  this component is already imported inside fastn.ftd remove all typography components from this file